relay-server/src/metrics_extraction/transactions/types.rs

```diff
@@ -1170,22 +1375,26 @@ mod tests {
         )
         .unwrap();
         metrics.retain(|m| m.name.contains("lcp"));
+        let mut tags = extract_universal_tags(event.value().unwrap(), &config);
+        tags.insert(
+            UniversalTags::Custom("satisfaction".to_owned()),
+            "frustrated".to_owned(),
+        );
+        tags.insert(UniversalTags::Platform, "other".to_string());
+        let measurement_tags = TransactionMeasurementTags {
+            measurement_rating: Some("good".to_string()),
+            universal_tags: tags,
+        };
+
         assert_eq!(
             metrics,
-            &[Metric::new_mri(
-                METRIC_NAMESPACE,
-                "measurements.lcp",
-                MetricUnit::Duration(DurationUnit::MilliSecond),
-                MetricValue::Distribution(41.0),
-                UnixTimestamp::from_secs(1619420402),
-                {
-                    let mut tags = BTreeMap::new();
-                    tags.insert("satisfaction".to_owned(), "frustrated".to_owned());
-                    tags.insert("measurement_rating".to_owned(), "good".to_owned());
-                    tags.insert("platform".to_owned(), "other".to_owned());
-                    tags
-                }
-            )]
+            &[TransactionMetric::Measurement {
+                name: "lcp".to_string(),
+                value: 41.0,
+                unit: MetricUnit::Duration(DurationUnit::MilliSecond),
+                tags: measurement_tags
+            }
+            .into_metric(UnixTimestamp::from_secs(1619420402))]
```

Comment:
Why construct the metric we test against via `TransactionMetric`, and not leave it as it was?

Alternatively, you can also use `assert_debug_snapshot!` instead of `assert_eq!` here.

Reply:
the `new_mri` function is changed so i couldnt leave it as it is, it seemed natural to change it to the new way, maybe theres something i dont understand here

Reply:
update: reverted the new_mri function so this is changed back too